### ISSUE TITLE
API schema not updated

### ISSUE THREAD

**Issue by lobbin** [a85/POSTMan-Chrome-Extension#10263]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

I'm adding a pipeline to update our API definition when it changes. I'm calling the update schema API, and yet, the definition is not updated when I look at the API from the web interface.

### Steps To Reproduce

```
  const updateSchemaBody = {
    schema: {
      language: 'yaml',
      type: 'openapi3',
      schema: fs.readFileSync('openapi.yml').toString()
    }
  }
  const updateSchema = await put(`/apis/${apiId}/versions/${apiVersionId}/schemas/${schemaId}`, JSON.stringify(updateSchemaBody))
```

Response:
```
{
  schema: {
    id: 'x',
    type: 'openapi3',
    language: 'yaml',
    apiVersion: 'y',
    createdAt: '2021-08-18T06:42:08.000Z',
    updatedAt: '2021-08-31T09:17:30.000Z',
    createdBy: 'z',
    updatedBy: 'z'
  }

```

### Environment Information

```markdown
- Operating System: Mac
- Platform Type: Postman API
- Postman Version: N/A
```

**Comment by xk0der** [comment]
@lobbin do you get any errors when hitting the update-schema endpoint?

With just the partial snippet it's hard to tell what could be going wrong.
If you haven't already, include some logging statements in your script, to see if the call to the update-schema endpoint is failing and with what error.

I would also suggest you try the API endpoint via the [Postman app](https://www.postman.com/downloads/) and see if things work fine in there.

**Comment by lobbin** [comment]
@xk0der No, I don't get any errors, that's why I posted the reply.

```
{
  schema: {
    id: 'x',
    type: 'openapi3',
    language: 'yaml',
    apiVersion: 'y',
    createdAt: '2021-08-18T06:42:08.000Z',
    updatedAt: '2021-08-31T09:17:30.000Z',
    createdBy: 'z',
    updatedBy: 'z'
  }
}
```

However, I also tested this directly via Postman and there it actually works. I'm using the npm module axios on our side, and it seems I made an error as now it works via my script as well.

But still, the API call tells me the API have been updated on your side when it hasn't, so something isn't fully error checked on your side either ;)
